core/src/SYCL/Kokkos_SYCL_ParallelFor_Team.hpp

```diff
@@ -164,10 +164,14 @@ class Kokkos::Impl::ParallelFor<FunctorType, Kokkos::TeamPolicy<Properties...>,
         m_league_size(arg_policy.league_size()),
         m_team_size(arg_policy.team_size()),
         m_vector_size(arg_policy.impl_vector_length()) {
-    // FIXME_SYCL optimize
```

Comment:
What was this comment referring to? Is it something these changes fix?

Reply:
In my opinion there is just nothing to fix anymore. We can of course try to improve `team_size_recommended` but the logic here is fine.